woocommerce/woocommerce
# Update blueprint export design

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



Update the current Blueprint export screen to the latest design.

Figma: Y5pUYSJPsGEud1vknUZhi8-fi-3814_18301


### Changes proposed in this Pull Request:

This PR applies the latest design for the "Export" section.




Closes #53675



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch and `blueprint` feature enabled.
2. Navigate to `WooCommerce -> Settings -> Advanced -> Blueprint`
3. Confirm the new design changes for the `Export` section. 
4. Select/De-select some items and click on `Export` button.
5. Inspect the schema and confirm your selections are included.



### Changelog entry












<details>



<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

This is under a feature-flag.
</details>
